src/main/resources/application-local.properties

```diff
@@ -18,12 +26,19 @@ lrs.samlurl=https://mocksaml.com/api/saml/metadata
 lrs.samlid=samltest
 
 brokerUrl=elrr-kafka:9092
+##brokerUrl=localhost:9092
+
 kafka.topic=test-1
 kafka.groupId=test-consumer-group
 
+##kafka.partitions=1
+kafka.partitions=6
```

Comment:
did you manage to confirm if local works the way it should? i.e. overwrites the other?